### ISSUE TITLE
data model: refactor `data-policies-v1.0.0.json`

### ISSUE THREAD

**Issue by tiborsimko** [cernopendata/opendata.cern.ch#1323]
`data-policies-v1.0.0` document type was added recently as an example of how the several record data types could be added.  It should be refined based on discussions in #1313. For example, we can end up having an abstract data type `publication` that would encompass both data policy papers and analysis papers...

**Comment by tiborsimko** [comment]
In the absence of alternative proposals, what about going for "documentation" following the storage policy?

**Comment by ioannistsanaktsidis** [comment]
I think ```documentation``` should do the job. cc @daslerr @ArtemisLav

**Comment by daslerr** [comment]
I agree with using `documentation` so as to match the storage policy.

**Comment by tiborsimko** [comment]
Update `documentation` schema following up the new record model. E.g. type/subtype, etc. The goal is to use the same fields.

Or simply remove, since data policy records are part of the record fixture.
